.github/workflows/default_branch_verification.yml

```diff
@@ -12,10 +12,10 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
-      - name: set up JDK 11
```

Comment:
Since it's only a name and to avoid unecessary changes, can we drop the value and just keep simple "set up JDK"?

Reply:
Right, done!